build-scripts/rcm-guest/push-to-mirrors.sh

```diff
@@ -83,12 +83,15 @@ chmod 775 "${PUDDLEDIR}/" || true
 # dereference the building symlink to the actual directory basename: e.g. "2017-06-09.4"
 LASTDIR=$(readlink --verbose "${PUDDLEDIR}/building")
 
+# use time of last data modification of building dir as suffix
+CHDT=$(stat -c %Y ${LASTDIR})
```

Comment:
Could the variable name be fully spelled out? It took me some time to arrive at a somewhat plausible guess what it would mean.

Reply:
changedate?

Reply:
I change it to MODT=modified timestamp

Reply:
Ah sorry, maybe i was not clear.

I also need to think while resolving this acronym. Could you spell the variable out so that it reads well? `change_date` or `modified_timestamp` or whatever does not make the human readers pause to figure out what it could mean.

Reply:
+1 on fully spelled out. `CHDT` and `MODT` aren't very clear.

Reply:
OK good suggestion